connector/semconvcheckerconnector/metadata.yaml

```diff
@@ -0,0 +1,13 @@
+type: semconvchecker
+scope_name: otelcol/semconvchecker
+
+status:
+  class: connector
+  stability:
+    development: [traces_to_logs, metrics_to_logs, logs_to_logs]
+  distributions: []
+  codeowners:
+    active: [madvikinggod, codeboten]
+
+tests:
+  config:
```

Comment:
not needed for the initial PR but it would be cool to have a metric generated by the component to track the number of semconv violations that it detects

Reply:
That shouldn't be too hard for something very simple at first pass.  Is there anything I need to change here to identify that I'm going to be exporting a counter?